src/components/EditableInput/EditableInput.tsx

```diff
@@ -164,7 +192,7 @@ const EditableInput = forwardRef(
           onArrowKeyDown &&
           (isArrowUpEvent(event) || isArrowDownEvent(event) || isArrowLeftEvent(event) || isArrowRightEvent(event))
         ) {
-          onArrowKeyDown(valueState, event);
+          onArrowKeyDown(event, valueState);
```

Comment:
changing the order of function prop parameters is a breaking change... I think it will be better not to do it